README.md

```diff
@@ -1,2 +1,157 @@
+✅ Async function — easy to integrate
+
+---
+
+## 📄 What does it do?
+
+Given a path to a `package.json`, what specifiers does it expose?
+
+The package export defaults to an `async function`.  
+It takes a path to a `package.json` as the only required argument.
+
+---
+
+## 📋 Output structure
+
+It resolves to an object with:
+
+- `name`: the package name
+- `version`: the package version
+- `engines`: the package's `engines` requirements
+- `problems`: a Set of strings describing problems or validation issues encountered during exports traversal.  
+  <sub>Note that these errors _do not_ necessarily interfere with the listed entry points being accessible at runtime.</sub>
+- `exports`: an object:
+  - `binaries`: Map of executable program names → file paths
+  - `latest`: the latest `[category][]` in the given Node version range
+  - `...categories`: each overlapping Node version category with:
+    - `import`: Map of import specifier → file path
+    - `require`: Map of require specifier → file path
+    - `files`: Set of relative file paths
+    - `tree`: Map of filenames & directories → their specifiers
+  - `pre-exports`: always present, whether in above list or not, with the above structure
+
+---
+
+## ⚙️ Options
+
+Besides the required `package.json` path, you can also pass an options object:
+
+- `node`: either `true` (reads `engines.node`), or a semver range string for Node versions to target. Defaults to the current Node version.
+
+---
+
+## 🖥️ CLI Usage Example
+
+You can also use the CLI tool located in:
+```
+./packages/ls-exports/bin/ls-exports
+```
+
+Example:
+```bash
+node ./packages/ls-exports/bin/ls-exports path ./packages/list-exports --json
+```
+
+---
+
+## 📄 API Example
+
+```js
+const listExports = require('list-exports');
+
+listExports('./package.json', { node: true })
+  .then((data) => {
+    console.log(JSON.stringify(data, null, 2));
+  })
+  .catch((err) => {
+    console.error(err);
+    process.exit(1);
+  });
+```
+
+---
+
+## 🧪 Running Tests
+
+To run the tests:
+```bash
+npm install
+npm test
+```
+
+Make sure tests pass before submitting a PR.
+
+---
+
+## 📁 Repository Structure
+
+- Root README with badges & examples
+- Source code in `/packages/list-exports/`
+- CLI code in `/packages/ls-exports/`
+- Tests & fixtures in `/packages/list-exports/tests/`
+- Configs: `.eslintrc`, `.eslintignore`, `.npmrc`, `.gitignore`, etc.
+
+---
+
+## 🙋‍♀️ Contributing
+
+We welcome contributions! Please:
+- Fork the repo and create your branch from `main`.
+- Run `npm install` to install dependencies.
+- Run tests with `npm test` before opening a PR.
+- Make sure your code is formatted with Prettier:
+  ```bash
+  npx prettier --write .
+  ```
+- Follow the existing code style and conventions.
+
+---
+
+## 🙌 Shoutouts
+
+Thanks to [Jordan Harband](https://github.com/ljharb) and all contributors for maintaining this project.
+
+---
+
+## 📜 License
+
+MIT © [Jordan Harband](https://github.com/ljharb)
+
+---
```

Comment:
this section isn't needed; github finds the license file on its own and displays it in various places.